### DISCUSSION TITLE
缺少对应 uid 的 Bilibili 用户登录后的 Cookie 值

### DISCUSSION THREAD

**Discussion by hfqi** [DIYgod/RSSHub#10765] [Q&A] [answered]
用vercel自建，想要订阅“用户关注动态”，官方文档里说要填入用户登录后的cookie，我已经能获取到完整的cookie了，但是填写之后还是提示缺少对应 uid 的 Bilibili 用户登录后的 Cookie 值。

请问具体填写cookie的格式是什么，下面是我填写的格式：
https://rssh-ub-wine-xi.vercel.app/bilibili/followings/dynamic/272975081?BILIBILI_COOKIE_272975081=[Cookie]

两串数字是我的uid，我把登录后的cookie替换掉[Cookie]了，第一次用github和rsshub，多多指教。

**Comment by TonyRL** [comment]
https://vercel.com/docs/concepts/projects/environment-variables

**Comment by hfqi** [comment, marked as answer]
已解决
在项目路径中添加.env配置文件，内容为BILIBILI_COOKIE_{UID}=[Cookie]，修改{UID}h和[Cookie]即可

**Comment by xjtuyan** [comment]
具体怎么操作呢? 在哪个路径下添加.env

**Reply by whitewatercn** [reply]
教程在 https://forum.beginner.center/t/topic/1541
